examples/guestbook/README.md

```diff
@@ -373,66 +408,90 @@ if (isset($_GET['cmd']) === true) {
 } ?>
 ```
 
-### Step Six: Create the guestbook service.
-
-Just like the others, you want a service to group your frontend pods.
-The service is described in the file `examples/guestbook/frontend-service.json`:
-
-```js
-{
-   "kind":"Service",
-   "apiVersion":"v1",
-   "metadata":{
-      "name":"frontend",
-      "labels":{
-         "name":"frontend"
-      }
-   },
-   "spec":{
-      "ports": [
-        {
-          "port":80
-        }
-      ],
-      "selector":{
-         "name":"frontend"
-      }
-   }
-}
+Note the use of the `redis-master` and `redis-slave` host names-- we're finding those services via the Kubernetes cluster's DNS service, as discussed above.
+
+### Step Six: Set up the guestbook frontend service.
+
+As with the other pods, we now want to create a service to group your frontend pods.
+The service is described in the file `examples/guestbook/frontend-service.yaml`:
```

Comment:
as we assume the user is already in example/guestbook/, use "frontend-service.yaml" directly.